backend/app/readers/create_readers.py

```diff
@@ -0,0 +1,30 @@
+from app.models.settings import Settings
+from app.readers.reader.reader import Reader
+from app.readers.reader.wikipedia_reader import WikipediaReader
+
+
+def create_readers(settings: Settings) -> tuple[Reader | None, ...]:
```

Comment:
This never returns a `None` reader anyway.

Reply:
Yes, I missed that. The return type of `create_readers` is now `tuple[Reader, ...]`.